docs/src/HOWTOs/howto-write-type-annotations.md

```diff
@@ -0,0 +1,347 @@
+# How to write type annotations
+
+**Warning:** *This HOWTO discusses how to write type annotations for the new
+type checker [Snowcat][], which has not been integrated with the model checker
+yet.  If you want to run the model checker, you have to write the [old type
+annotations][], until we release the integration (soon!)* :(
+
+This HOWTO gives you concrete steps to extend TLA+ specifications with type
+annotations. You can find the detailed syntax of type annotations in
+[ADR002][]. The first rule of writing type annotations:
+
+*Do not to write any annotations at all, until the type checker [Snowcat][] is
+asking you to write a type annotation.*
+
+Snowcat is quite smart, and it will most probably find most of the types for
+you.
+
+## Recipe 1: Recipe variables
+
+Consider the example [HourClock.tla][] from [Specifying Systems][]:
+
+```tla
```

Comment:
Should you maybe use the include syntax that we use elsewhere, to ensure that the manual? It helps ensure that the docs will exactly match whatever is in the source file: 

https://github.com/informalsystems/apalache/blob/shon/integration-521-rebase-test/docs/src/apalache/example.md#L6-L7

Reply:
Sure, that's nice. What is 55 here?

Reply:
```
{{#include ../../../test/tla/y2k.tla::55}}
```

Reply:
55 is the line number to start the inclusion from! You can also give a line number to end on. See the mdBook docs for more details :)

Reply:
Ah, that's nice!